config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -101,7 +101,7 @@ presubmits:
         - --scenario=kubernetes_e2e
         - --
         - --deployment=node
-        - --gcp-project=k8s-c8d-pr-node-e2e
```

Comment:
We don't have enough capacity in this pool to handle presubmits, can you try with a ci- job that uses this project first?

Reply:
@spiffxp hmm I don't see any other jobs that use this specific project

Reply:
We could try this one in `cri-containerd-node-e2e` https://testgrid.k8s.io/sig-node-containerd#containerd-node-conformance